x-pack/plugins/ml/public/plugin.ts

```diff
@@ -232,6 +232,14 @@ export class MlPlugin implements Plugin<MlPluginSetup, MlPluginStart> {
             } = await import('./register_helper');
             registerSearchLinks(this.appUpdater$, fullLicense, mlCapabilities, !this.isServerless);
 
+            if (pluginsSetup.triggersActionsUi) {
```

Comment:
Do we definitely want to be removing the capability check here? 
Do we want a user who doesn't have permission to view ML jobs to be able to see this?

Reply:
If the license is basic all these capabilities are disabled as well, that's why I removed the check. 
We can consider having a more granular check `(fullLicense && mlCapabilities.canUseMlAlerts && mlCapabilities.canGetJobs) || !fullLicense`. WDYT?

Reply:
Yeah, I think we should still be checking capabilities here.
We also use capabilities for controlling which features to show in serverless. If the AD feature is disabled, `canGetJobs` will be false.

Reply:
Updated in https://github.com/elastic/kibana/pull/173644/commits/c11b616a2d609999934c815c8d28e0cebb3dc4c8